ui/apps/platform/src/Containers/Administration/Events/AdministrationEventsTable.tsx

```diff
@@ -14,37 +14,49 @@ import IconText from 'Components/PatternFly/IconText/IconText';
 import { UseURLSortResult } from 'hooks/useURLSort';
 import {
     AdministrationEvent,
+    hasAdministrationEventsFilter,
     lastOccurredAtField,
     numOccurrencesField,
 } from 'services/AdministrationEventsService';
+import { SearchFilter } from 'types/search';
 
 import { getLevelIcon, getLevelText } from './AdministrationEvent';
 import AdministrationEventHintMessage from './AdministrationEventHintMessage';
 
 import './AdministrationEventsTable.css';
+import AdministrationEventsEmptyState from './AdministrationEventsEmptyState';
+
+const colSpan = 6;
```

Comment:
I noticed when you extracted this out, its value went from 5 to 6. But I'm not sure why?

Reply:
Good eyes. Probably copy and paste from where I found an example of empty state.